web-client/integration-tests/admissionsClerkPractitionerJourney.test.ts

```diff
@@ -99,7 +99,7 @@ describe('admissions clerk practitioner journey', () => {
   loginAs(cerebralTest, 'admissionsclerk@example.com');
   admissionsClerkAddsPractitionerEmail(cerebralTest);
 
-  // show Practitioners only, and not case-users (bug ref: #8081)
+  // // show Practitioners only, and not case-users (bug ref: #8081)
```

Comment:
probably unnecessary; might be able to just get rid of the line entirely at this point.

Reply:
Good point, thanks.